.gitlab/update-openjdk.sh

```diff
@@ -31,6 +31,7 @@ create_collector_pr() {
     if [[ -n "$version" ]]; then
       echo ">>> Updating openjdk version to $version ..."
       sed -i "s|^ARG JDK_VERSION=.*|ARG JDK_VERSION=${version}|" internal/signalfx-agent/bundle/Dockerfile
+      sed -i "s|^ARG JDK_VERSION=.*|ARG JDK_VERSION=${version}|" cmd/otelcol/Dockerfile.windows
```

Comment:
```suggestion
      sed -i "s|^ARG JDK_VERSION=.*|ARG JDK_VERSION=\"${version}\"|" cmd/otelcol/Dockerfile.windows
```
Alternatively, remove the quotes from `Dockerfile.windows` for the arg.  Otherwise, this script will open PRs just for the quotes.